database_email_backend/admin.py

```diff
@@ -43,7 +43,7 @@ def file_link(self, obj):
 
 
 class EmailAdmin(admin.ModelAdmin):
-    list_display = ('from_email', 'to_emails', 'subject', 'body_stripped', 'sent_at', 'attachment_count')
+    list_display = ('from_email','reply_to', 'to_emails', 'subject', 'body_stripped', 'sent_at', 'attachment_count')
```

Comment:
Looks like there's a missing space before 'reply_to'.  If you're feeling ambitious, that whole file could use some pep8 love.